### ISSUE TITLE
[feature request] Add hotkey for Fake popup

### ISSUE THREAD

**Issue by yakimka** [tigersoftdev/faker-intellij-plugin#9]
Hello, thank you for this plugin.

Would it be possible to add a popup hotkey?

For example like in String Manipulation plugin:
![image](https://user-images.githubusercontent.com/28621349/105077852-c82dbe00-5a95-11eb-9ccf-c332c9b43a02.png)

**Comment by tigersoftdev** [comment]
Hello, thank you @yakimka for this request, this might be very handy if someone uses plugin often. I will have to make some maintanace work and will look at this feature, but I can't tell the exact time this will be added. Feel free to work on it on your own if you want, we could merge it to the main repo as soon as you issue a PR.

**Comment by AbdelkaderBah** [comment]
@tigersoftdev I am sure you are busy, Faker is a great enhancement to most developer workflow.

[feature request] Add hotkey
[feature request] Make faker work on Multiple lines

Thank you for your great work.

**Comment by tigersoftdev** [comment]
> 
> 
> @tigersoftdev I am sure you are busy, Faker is a great enhancement to most developer workflow.
> 
> [feature request] Add hotkey
> [feature request] Make faker work on Multiple lines
> 
> Thank you for your great work.

@AbdelkaderBah thank you for your appreciation. How do you imagine faker to work with multiple lines?

**Comment by tigersoftdev** [comment]
@yakimka, @AbdelkaderBah have you tried `ALT + Insert` or its Mac equivalent? This should trigger popup to show:
![hotkey](https://user-images.githubusercontent.com/47889353/112024461-845a4200-8b34-11eb-8ca9-94be6832522a.png)

**Comment by yakimka** [comment]
Yes, this works, but I can't search in "Actions" by fake data type like in other plugins
![image](https://user-images.githubusercontent.com/28621349/112172539-78868280-8bfd-11eb-892f-9736340300cf.png)

**Comment by tigersoftdev** [comment]
I was able to find this action typing `Fake`
![action](https://user-images.githubusercontent.com/47889353/112176798-b208bf00-8bf8-11eb-9db9-881c29353b1c.png)

**Comment by yakimka** [comment]
Yes, but you can't find any nested action like Url, Credit card, City, etc.

![image](https://user-images.githubusercontent.com/28621349/112191898-23ec0300-8c0f-11eb-8504-5c0945e44c57.png)

**Comment by tigersoftdev** [comment]
Well, I'm not sure if this will be possible. I assume that other plugins have their actions/options defined explicitly in xml config file (usually there is not many of them) and because of that they get indexed as potential action and you can search them easily. Faker only defines entry point in that place, other nested submenus/actions are built "on the fly" using Reflecion API, which may cause that they're not included in global action list. Manual definig faker actions would be painful, there are many of them as you probably noticed ;)
